<launch>
  <rosparam command="load" file="$(find manipulator_description)/param/dhparams.yaml" />
  <node pkg="manipulator_description" name="dhtorpy" type="dhtorpy_server"/>
</launch>
